client/src/rubin/nublado/client/_mock.py

```diff
@@ -689,8 +689,7 @@ async def _handle_progress(self, request: Request, user: str) -> Response:
             )
         elif state == MockJupyterState.LAB_RUNNING:
             body = (
-                'data: {"progress": 100, "ready": true, "message": "Ready"}\n'
-                "\n"
+                'data: {"progress": 100, "ready": true,"message": "Ready"}\n\n'
```

Comment:
This change is wrong (it makes the line 80 columns and our line length limit is 79). I think that's because of the recursion, because now Ruff isn't configured because the configuration lives only at the top level and it's now using its default of 120 or 100 columns or whatever it is.

This change should go away once Ruff is fixed to run only from the top level where all the configuration is.

Reply:
Hm.  My editor shows that at 79 cols.

<img width="870" height="337" alt="Image" src="https://github.com/user-attachments/assets/935a68eb-d532-4ddc-8260-4e75e65e03be" />

Reply:
But then the one below that should be too?!?